src/routes/safe/components/Apps/components/AppFrame.tsx

```diff
@@ -101,6 +97,12 @@ const AppFrame = ({ appUrl }: Props): ReactElement => {
   const errorTimer = useRef<number>()
   const [, setAppLoadError] = useState<boolean>(false)
 
+  const safeAppsRpc = getSafeAppsRpcServiceUrl()
```

Comment:
why move it outside the hook instead of adding it as a dep? the ref is stable

Reply:
Linting issues. It doesn't complain now.